kronolith/lib/Event.php

```diff
@@ -2855,6 +2890,9 @@ public function readForm(Kronolith_Event $existing = null)
                 if (!isset($newattendees[$email])) {
                     unset($attendees[$email]);
                 }
+                if (Kronolith::isUserEmail($this->creator, $email)) {
+                    $attendees[$email]['response'] = Horde_Util::getFormData('attendance');
+                }
             }
         }
```

Comment:
Actually, ignore that. I was getting confused at having both the Status and Attendance drop down menus visible. I'm guessing the idea was to show the status that was sent back to the organizer in one field, and allow the attendee to send a change by editing the second field? This is a bit confusing, IMO. Maybe it would be better if instead of just disabling the "Status" field, we put some text saying something like <em>You have responded that you will attend this event.</em> or <em>You have not yet indicated your attendance to this event.</em>

Thoughts?